model/PermissionsService.php

```diff
@@ -163,8 +161,35 @@ private function dryAdd(array $add, core_kernel_classes_Resource $resource, arra
         }
     }
 
-    private function getResourcesToUpdate(core_kernel_classes_Resource $resource, bool $isRecursive): array
+    /**
+     * It is likely the strategy has not checked for duplicate permissions;
```

Comment:
Same as here, lets please remove unnecessary comments. If you are deduplicating, means for some reason, the code might be duplicated and you need to filter it. This reason can change, and your comment is obsolete.

Same here https://github.com/oat-sa/extension-tao-dac-simple/pull/199/files#r985070633

Reply:
I've removed it in ee1ae41942e99d6bcb5572b0b5d2084b178b0a77